pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -2584,10 +2618,15 @@ private GraphChromatogram CreateGraphChrom(string name)
             graphChrom.ChangedPeakBounds += graphChromatogram_ChangedPeakBounds;
             graphChrom.PickedSpectrum += graphChromatogram_PickedSpectrum;
             graphChrom.ZoomAll += graphChromatogram_ZoomAll;
-            _listGraphChrom.Add(graphChrom);
+            AddGraphChromToList(graphChrom);
             return graphChrom;
         }
 
+        private void AddGraphChromToList(GraphChromatogram graphChrom)

```

Comment:
Why this change? It doesn't seem to get used throughout the code, and it doesn't seem to provide any encapsulation, being private.